README.md

```diff
@@ -151,7 +165,7 @@ vi ../../samples/SamplesCommon/samples_common.h # modify connection string to in
 
 Please note the current build script is only tested on Ubuntu 16.04. Please update the script accordingly for other distributions.
 
-Please note that starting from 2.10.0, Casablanca requires a minimum version of CMake v3.1, so the default CMake on Ubuntu 14.04 cannot support Casablanca build. User can upgrade CMake by themselves to build Casablanca.
+Please note that starting from 2.10.0, Casablanca requires a minimum version of CMake v3.1, so the default CMake on Ubuntu 14.04 cannot support Casablanca build. User can upgrade CMake by themselves to build Casablanca. 5.0.1 is the last version that uses Casablanca 2.9.1, which supports default Ubuntu 14.04's CMake.
```

Comment:
last version or latest version?

Reply:
Actually, this is mis-leading. This line is refined.